platform.local.txt

```diff
@@ -0,0 +1,2 @@
+compiler.c.extra_flags=-DASYNC_TCP_SSL_ENABLED=1 -DASYNC_TCP_SSL_BEARSSL=1 -DESPZW_DEBUG_LEVEL=1 -DESPZW_LOG_TIME=1

```

Comment:
Did you really mean to include this file?

Reply:
LOL, accident... maybe I should add this file to .gitignore?